app/views/devise/registrations/edit.html.erb

```diff
@@ -1,43 +1,62 @@
 <h2>Edit <%= resource_name.to_s.humanize %></h2>
+<% if @user.image.attached? %>
+  <image class="rounded-circle mx-auto d-block" width= "100px" height="100px" src="<%=(url_for(@user.image)) %>">
+<% end %>
+<% if @user.image.filename != "default-avatar.jpg" %> 
+  <%= button_to 'Remove Image', user_registration_path(@user,
+    user: { deleteImage: "1" }),
+    method: :patch,
+    remote: true,
+    data: { confirm: "Are you sure?" },
+    class: 'btn btn-danger btn-sm m-1 mx-auto d-block'  %>
+<% end %>
+<h6 class="text-center">User name: <strong><%= @user.user_name %><strong></h6>
 
-<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
+<div class="d-flex flex-column text-center">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
-  </div>
+     
+    <div class="mb-3 field">
+      <%= f.file_field :image, placeholder: "User profile photo" %>
+    </div> 
```

Comment:
Also you can remove this white spaces

Reply:
:+1:  Thanks Jorge! i will review the empty spaces